.github/workflows/pythonapp-min.yml

```diff
@@ -98,6 +98,9 @@ jobs:
       with:
         path: ${{ steps.pip-cache.outputs.dir }}
         key: ubuntu-latest-latest-pip-${{ steps.pip-cache.outputs.datew }}
+    - name: install prerelease and py311
+      if: matrix.python-version == '3.11'
+      run: pip install --pre torch --extra-index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
Is there no GPU available for these tests?

Reply:
no, this is the minimum cpu tests, I think this should first work fine because we invest GPU resources.

Reply:
Ok, great. On the pytorch website they mention that some features like JIT aren't there yet, so maybe we could decorate those tests to skip if python version == 3.11. Other tests I think we can just fix so that they work with all versions of python3.

Reply:
Or maybe even better we can figure out whether jit works for a given version and store as global variable:
```python
try:
    # try to do some jitting
    CAN_JIT = True
except SomeException:
    CAN_JIT = False

class TestSomething(TestClass):
    @skipUnless(CAN_JIT)
    def test_something_with_jit(self, ...):
        pass
```

Reply:
the only remaining error is this one
```py
  File "/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/torch/serialization.py", line 1136, in _load
    result = unpickler.load()
             ^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/torch/_tensor.py", line 70, in _rebuild_from_type_v2
    ret = torch._utils._set_obj_state(ret, state)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.11.0/x64/lib/python3.11/site-packages/torch/_utils.py", line 369, in _set_obj_state
    for k, v in dict_state.items():
                ^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'items'

```

any idea?

Reply:
If `torch.save` doesn't successfully save anything, then maybe `torch.load` will return `None`?

Reply:
the stable release version works fine...